README.md

```diff
@@ -86,7 +86,7 @@ $ nix-build -j4 -A server -o server-out
 
 ### Get a GHC based development shell
```

Comment:
I think it would help to have a line clarifying what is meant by "development shell", and why you need it.
If I understand correctly
* It gives you a shell prompt where all the dependencies installed by nix are available. 
* A bunch of the other commands need to be run within the dev shell.